controllers/ovndbcluster_controller.go

```diff
@@ -86,6 +87,7 @@ func (r *OVNDBClusterReconciler) GetLogger(ctx context.Context) logr.Logger {
 }
 
 //+kubebuilder:rbac:groups=ovn.openstack.org,resources=ovndbclusters,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=ovn.openstack.org,resources=ovncontroller,verbs=create;update;patch;delete
```

Comment:
do we really need to create/update/delete?

Reply:
Yeah.
Creation is needed bc we need to add the OVNEncapType once OVNController recreated
Update bc it could happen that user modifies the ExternalIDs.EncapType and configMap needs to update value
Delete is needed bc we want to remove OVNEncapType from config map

Reply:
But these rules are for ovncontroller not configmap, and ovndbcluster controller don't need to create/delete/update ovncontrollers.

Reply:
Oh, true, I miss understood the verbs, changing it, thanks for noticing guys :)